### PR TITLE
reference refine the ecosystem tools user guide

### PR DESCRIPTION
**Pull request by csuzhangxc**
### What is changed, added or deleted? (Required)

- update the ecosystem tools user guide to make it clearer.
    - replace BR with Dumpling & Lightning compare with #3497
- copy _export-or-backup-using-dumpling.md_ from master branch
    - but remove the content about BR.

### Which TiDB version(s) do your changes apply to? (Required)

- [ ] master (the latest development version)
- [ ] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [x] v3.0 (TiDB 3.0 versions)
- [x] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

#3497 
#3151

### CODE REVIEW

**Comment by GregoryIan** [comment]
LGTM

File: ecosystem-tool-user-guide.md [hunk 1]
```diff
@@ -1,135 +1,81 @@
 ---
-title: TiDB 生态工具使用指南
+title: TiDB 生态工具功能概览
 category: reference
-aliases: ['/docs-cn/v3.0/reference/tools/user-guide/','/docs-cn/v3.0/how-to/migrate/from-mysql/', '/docs-cn/v3.0/how-to/migrate/incrementally-from-mysql/', '/docs-cn/v3.0/how-to/migrate/overview/', '/docs-cn/v3.0/reference/tools/use-guide/']
+aliases: ['/docs-cn/dev/reference/tools/user-guide/','/docs-cn/dev/how-to/migrate/from-mysql/', '/docs-cn/dev/how-to/migrate/incrementally-from-mysql/', '/docs-cn/dev/how-to/migrate/overview/', '/docs-cn/dev/reference/tools/use-guide/']
```
Comment (yikeke) [inline ecosystem-tool-user-guide.md:4]:
```suggestion
aliases: ['/docs-cn/v3.0/reference/tools/user-guide/','/docs-cn/v3.0/how-to/migrate/from-mysql/', '/docs-cn/v3.0/how-to/migrate/incrementally-from-mysql/', '/docs-cn/v3.0/how-to/migrate/overview/', '/docs-cn/v3.0/reference/tools/use-guide/']
```

File: export-or-backup-using-dumpling.md [hunk 2]
```diff
@@ -0,0 +1,114 @@
+---
```
Comment (yikeke) [inline export-or-backup-using-dumpling.md:1]: 这篇文章需要加到 TOC

File: ecosystem-tool-user-guide.md (hunk 1 above)
Comment (csuzhangxc) [reply ecosystem-tool-user-guide.md]: 感谢提醒，我以后注意

File: export-or-backup-using-dumpling.md (hunk 2 above)
Comment (csuzhangxc) [reply export-or-backup-using-dumpling.md:1]: 在 https://github.com/pingcap/docs-cn/pull/3519/commits/15ddb6d0ad3552efcfb7c861dfcb33d01147da0e 加到 TOC 了

**Pull request merged**